src/components/layout/BlobBox.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const outputBlobs = (blobs: any) =>
+  blobs.map(
+    (blob: any, bi: number) =>
+      `
+      &.blob-${blob.id}-${bi} {
```

Comment:
The problem in Chrome was that `translate` isn't supported by webkit browsers as a property itself, so you have to use it as a function inside of transform. Also, transform is read right-to-left, so putting the transform stuff first before the translate function wouldn't work. Anyway, all fixed now 👍🏻 @Texian